templates/pkg/resource/sdk.go.tpl

```diff
@@ -145,7 +145,7 @@ func (rm *resourceManager) sdkDelete(
 		exit(err)
 	}()
 
-{{ if .CRD.CustomDeleteMethodName }}
+{{- if .CRD.CustomDeleteMethodName }}
```

Comment:
I just made a tiny change to the custom delete method. Please make sure that is working fine acmpca-controlelr @ndbhat